src/test/java/net/snowflake/client/jdbc/ConnectionWithOCSPModeIT.java

```diff
@@ -440,6 +445,49 @@ public void testWrongHost() throws InterruptedException {
     fail("All retries failed");
   }
 
+  /** Test connectivity with disableOCSPChecksMode and insecure mode enabled. */
+  @Test
+  public void testDisableOCSPChecksModeAndInsecureMode() throws SQLException {
```

Comment:
Also worth having tests for `disableOCSPChecks` only and `insecureMode` only.

Reply:
These tests are already added in ConnectionIT class.